ZuluContent/Zulu/Engines/Magic/EnchantmentDictionary.cs

```diff
@@ -229,6 +228,50 @@ public void SetFromResourceType(Type enchantmentType, int value)
             }
         }
 
+        public void SetFromFortified(IEnchantmentValue value)
```

Comment:
With SRP in mind probably shouldn't be putting these methods on to the `EnchantmentDictionary`, it's purpose is to just store values not have business logic from Fortifying. Also you don't really need this, since you're deleting the helm after just transfer the enchantments with `EnchantmentDictionary.Set`